CONTRIBUTING.md

```diff
@@ -0,0 +1,57 @@
+# Contributing Guidelines
+
+Thank you for considering contributing to this package. Please take a moment to review [Storyblok's general contributing guidelines](https://github.com/storyblok/.github/blob/main/CONTRIBUTING.md) if it's your first time to find info on submitting PR or issues.
+
+## Contributing to the code
+
+Find useful these guidelines and set of recommendations to help improving this library.
+
+### Setup
+
+This package relies on [pnpm workspaces](https://pnpm.io/workspaces) to manage dependencies. For instructions on how to install pnpm, please visit [pnpm.io](https://pnpm.io/installation).
+
+```bash
+pnpm install
+```
+
+### Build
+
+```bash
+pnpm build
+```
+
+### Development
+
+We strongly recommend using the `playgrounds` within the package to develop and manually test the package.
```

Comment:
Making this comment for all READMEs, let's not use playgrounds in code notation if it's not an API or function. Let's do bold or italic if you want highlight or even a link to the folder in here could work.

Reply:
That's a good point actually

Reply:
@jeremenichelli what about files, often we use `package.json` to highlight a file in the codebase

Reply:
Those are fine, it's a convention over GitHub and also the extension signals to the user is a file already. All good with using code there.